src/templates/src/index.ts.squirrelly

```diff
@@ -1,17 +1,35 @@
 import {
+{{ @if (it.aptitudes.includes("clipboard")) }}
   clipboard,
+{{ /if }}
+{{ @if (it.aptitudes.includes("filesystem")) }}
   filesystem,
+{{ /if }}
+{{ @if (it.aptitudes.includes("keyboard")) }}
   keyboard,
+{{ /if }}
+{{ @if (it.aptitudes.includes("network")) }}
   network,
+{{ /if }}
+{{ @if (it.aptitudes.includes("ui")) }}
   ui,
+{{ /if }}
   whisper,
+{{ @if (it.aptitudes.includes("window")) }}
   window,
+{{ /if }}
 } from "@oliveai/ldk";
```

Comment:
If there are no aptitudes chosen, would this lead to an empty import (that's broken I guess)?

Reply:
I left the `whisper` import alone so if nothing is selected that still is imported and nothing breaks. I just left it in alphabetical order so it looks like it's lumped in with the other ifs